service/src/test/java/uk/nhs/adaptors/gp2gp/ehr/mapper/SpecimenMapperTest.java

```diff
@@ -0,0 +1,125 @@
+package uk.nhs.adaptors.gp2gp.ehr.mapper;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.Mockito.lenient;
+import static org.mockito.Mockito.when;
+
+import java.io.IOException;
+import java.util.List;
+import java.util.stream.Stream;
+
+import org.hl7.fhir.dstu3.model.Observation;
+import org.hl7.fhir.dstu3.model.Reference;
+import org.hl7.fhir.dstu3.model.ResourceType;
+import org.hl7.fhir.dstu3.model.Specimen;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.mockito.stubbing.Answer;
+
+import uk.nhs.adaptors.gp2gp.common.service.FhirParseService;
+import uk.nhs.adaptors.gp2gp.ehr.mapper.diagnosticreport.ObservationMapper;
+import uk.nhs.adaptors.gp2gp.ehr.mapper.diagnosticreport.SpecimenMapper;
+import uk.nhs.adaptors.gp2gp.utils.ResourceTestFileUtils;
+
+@ExtendWith(MockitoExtension.class)
+public class SpecimenMapperTest {
+
+    private static final String TEST_FILE_DIRECTORY = "/ehr/mapper/specimen/";
+    private static final String DIAGNOSTIC_REPORT_DATE = "2020-10-12";
+    private static final String INPUT_OBSERVATION_RELATED_TO_SPECIMEN = "input-observation-1.json";
+    private static final String INPUT_OBSERVATION_NOT_RELATED_TO_SPECIMEN = "input-observation-2.json";
```

Comment:
Can we rename the files to something more descriptive? e.g. `input-observation-related-to-specimen.json`